setup.cfg

```diff
@@ -48,7 +48,7 @@ install_requires =
     preshed>=3.0.2,<3.1.0
     thinc>=8.0.12,<8.1.0
     blis>=0.4.0,<0.8.0
-    wasabi>=0.8.1,<1.1.0
+    wasabi>=0.9.0,<1.1.0
```

Comment:
You need to make sure that the version is consistent with the one from `requirements.txt` - this is actually why the test `test_build_dependencies` fails.